iceoryx_posh/source/runtime/message_queue_interface.cpp

```diff
@@ -223,6 +223,11 @@ MqRuntimeInterface::MqRuntimeInterface(const std::string& roudiName,
     , m_AppMqInterface(appName)
     , m_RoudiMqInterface(roudiName)
 {
+    if (!m_AppMqInterface.isInitialized()) {
+        errorHandler(Error::kMQ_INTERFACE__UNABLE_TO_CREATE_APPLICATION_MQ);
```

Comment:
I have no idea how to test this. Especially with the unix domain socket on mac, which is the reason for this. This can only happen if the unix domain socket cannot be created, e.g. when the path name is too long. But that is already taken care of with `MAX_PROCESS_NAME` and there is already a test in PoshRuntime for MAX_PROCESS_NAME